src/BarrelCalorimeterSciFi_geo.cpp

```diff
@@ -411,4 +411,4 @@ vector<FiberGrid> gridPoints(int div_n_phi, double div_dr, double trd_x1, double
   return results;
 }
 
-DECLARE_DETELEMENT(epic_EcalBarrelInterlayers, create_detector)
+DECLARE_DETELEMENT(epic_EcalBarrelSciFi, create_detector)
```

Comment:
```suggestion
DECLARE_DETELEMENT(epic_EcalBarrelScFi, create_detector)
```
I thought, this was the current naming convention.

Reply:
(It's okay to change it, but we need to see where else to rename)

Reply:
Not two days before a release please 🙏

Reply:
We can keep ScFi now, but decide on adopting SciFi for August.